source/codegen/metadata_mutation.py

```diff
@@ -176,7 +176,12 @@ def populate_grpc_types(parameters, config):
             service_class_prefix)
 
 
-def add_attribute_values_enums(enums, attribute_enums_by_type, service_class_prefix):
+def get_short_enum_type_name(type):
```

Comment:
Note: I'll probably have to iterate on this method. Especially for arrays. But this works reasonably well. A good alternative could be to take "everything after the word Attribute" from the method name and use that as a starting point.

Sample DAQ input: `int32`
Sample MI input: `ViInt32`
Desired output in both cases: `Int32`